components/app.js

```diff
@@ -28,7 +36,37 @@ class App{
             error: this.handleGetGradesError
         })
     }
+
     start(){
         this.getGrades();
+        this.gradeForm.onSubmit(this.createGrade);
+    }
+
+    createGrade(name, course, grade){
+        $.ajax({
+            method: "POST",
+            url: "https://sgt.lfzprototypes.com/api/grades",
+            data:
+            {
+                "name": name,
+                "course": course,
+                "grade": grade
+            },
+            headers: { "X-Access-Token": "4zYBwVu7" },
+            success: () => {
+                this.handleCreateGradeSuccess()
+            },
+            error: () => {
```

Comment:
Rather than using an anonymous function with a function call within it here, you should use a callback function 👍